docs/spec.md

```diff
@@ -588,6 +588,13 @@ If you try to concatenate two literals of different types such as
 `arr := [1 2] + ["a" "b"]`, you will get a parse error. Use
 `arr := [1 2 "a" "b"]` instead.
 
+Arrays can be repeated with the `*` operator, for example `[0] * 5`
+results in `[0 0 0 0 0]`. `["hello" "world"] * 2` results in `["hello"
+"world" "hello" "world"]`. An array repeated zero times results in an
+empty array of the same type as the array. The number on the right hand
+side of the `*` operator must be an non-negative integer value. Using a
+decimal value will result in an error.
```

Comment:
Using a decimal or non-integer value will result in an error (see below)?

Reply:
what's below? I will change to "non-integer or negative"

Reply:
done. I guess "see below" refers to the comment above? Let me know if you think that reads ok now.

Reply:
yeah, sorry, the below was below when I made the comment.... you read it correctly.
🍍

Reply:
yeah, sorry, the below was below when I made the comment.... you read it correctly.
🍍

Reply:
Understood. See above.